source/agora/node/Validator.d

```diff
@@ -302,11 +302,11 @@ public class Validator : FullNode, API
         auto signed_validators = BitMask(block.header.validators.count);
         signed_validators.copyFrom(block.header.validators);
 
-        auto node_validator_index = this.nominator.enroll_man
-            .getIndexOfValidator(block.header.height, this.config.validator.key_pair.address);
-
+        const self_utxo = this.enroll_man.getEnrollmentKey();
+        auto validators = this.ledger.getValidators(block.header.height);
+        const ptrdiff_t node_validator_index = validators.countUntil!(v => v.utxo() == self_utxo);
```

Comment:
First time I've seen this one. Looks useful.

Reply:
I wish it was named `indexOf` though...